.travis.yml

```diff
@@ -41,24 +41,24 @@ jobs:
       stage: pre-test
       script: ./scripts/ci/ci_run_all_static_checks.sh
       env: >-
-        PYTHON_VERSION=3.6
+        PYTHON_SHORT_VERSION=3.6
```

Comment:
Is there a reason we renamed PYTHON_VERSION to PYTHON_SHORT_VERSION?

Reply:
Yes. PYTHON_VERSION is set by default by the python base image and it is the full version  (PYTHON_VERSION=3.7.9). It got so confusing that I decided I finally have to separate those two.

Reply:
Sounds good

Reply:
What do you think about renaming to ``PYTHON_MINOR_VERSION``?

Reply:
PYTHON_MAJOR_MINOR_VERSION is better